src/dataflow/components/dataflow-program.tsx

```diff
@@ -160,13 +164,43 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
         if (change) {
           (async () => {
             await engine.abort();
-            await engine.process(editor.toJSON());
+            await engine.process(this.programEditor.toJSON());
           })();
         }
       });
 
-      editor.view.resize();
-      (editor as any).trigger("process");
+      this.programEditor.view.resize();
+      (this.programEditor as any).trigger("process");
```

Comment:
Is this cast necessary?

Reply:
Doesn't appear to be needed.  But I better test a little to ensure proper behavior.

Reply:
Fixed.